tests/phpunit/WebTest/Contribute/UpdateContributionTest.php

```diff
@@ -262,9 +262,9 @@ public function testChangePaymentInstrument() {
     $this->waitForAjaxContent();
     //Assertions
     $subtractedTotal = $this->_getPremiumActualCost($contId, NULL, 'Payment Processor Account');
-    $this->assertEquals($subtractedTotal, -$amount, "Verify amount deleted from old account");
+    $this->assertEquals($subtractedTotal, NULL, "Verify amount deleted from old account");
```

Comment:
‎I don't think it's a great fix to check NULL. Could you try adding 5th parameter as entity table to _getPremiumActualCost functionSomething like$subtractedTotal = $this->_getPremiumActualCost($contId, NULL, 'Payment Processor Account‎', NULL, 'civicrm_contribution');                                                                                                                                     Or set $entityTable = 'civicrm-contribution'‎ by default in function _getPremiumActualCost.                                                                                                                                     HTHPradeepSent from my BlackBerry 10 smartphone.